utils.py

```diff
@@ -287,9 +300,21 @@ def get_runmode():
     else:
         return 'script'
 
-def install_packages(*packages):
-    command = [config.SUPERUSER_COMMAND, config.PACKAGE_MANAGER_COMMAND] + list(packages)
-    subprocess.run(command, check=True)
+def query_packages(packages):
+    missing_packages = [ ]
+    
+    for p in packages:
+        command = f"{config.SUPERUSER_COMMAND} {config.PACKAGE_MANAGER_COMMAND_QUERY}{p}"
+        result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True, text=True)
+        if result.returncode != 0:
+            missing_packages.append(p)
+
+    return missing_packages
+
+def install_packages(packages):
+    command = f"{config.SUPERUSER_COMMAND} {config.PACKAGE_MANAGER_COMMAND_INSTALL} {' '.join(packages)}"
+    print("COMMAND: ", command)
+    subprocess.run(command, shell=True, check=True)
```

Comment:
maybe you used `print(` here during testing and intended to remove it, but if you want to keep it you should switch it to `logging.info(` to log the command.